### DISCUSSION TITLE
How to change color of the map?

### DISCUSSION THREAD

**Discussion by BenJackGill** [inocan-group/vue3-google-map#221] [Q&A] [answered]
I want to change the color of the map itself (but not my custom UI elements on top of the map) and so I tried using the [MapTypeStyle](https://developers.google.com/maps/documentation/javascript/reference/map#MapTypeStyle) prop but I don't think it is supported because it had no effect.

```Vue
<GoogleMap
  :api-key="apiKey"
  class="google-map"
  style="width: 100%; height: 500px"
  map-type-id="terrain"
  map-type-style="[{ 'stylers': [{ 'color': '#FF0000' }] }]"
  :center="{ lat: 39.9511163, lng: -75.1651992 }"
  :zoom="15"
  disable-default-ui
  disable-double-click-zoom
  :keyboard-shortcuts="false"
  gesture-handling="none"
>
  <rank-map-grid-circle
    v-for="circle in circles"
    :key="circle.id"
    :lat="circle.lat"
    :lng="circle.lng"
  />
</GoogleMap>
```

I also tried using inline styles like `style="width: 100%; height: 500px; color: #FF0000"` and also a similar try with a CSS class but that did not work.

Is there another way to achieve this?

Maybe MapTypeStyle should be supported?

**Comment by HusamElbashir** [comment, marked as answer]
If you're asking about theming there's a `styles` prop you can use. You can also use some of the bundled themes. See:
https://github.com/inocan-group/vue3-google-map/issues/104#issuecomment-1265210312
